src/patternfly/components/Popover/docs/code.md

```diff
@@ -1,19 +1,31 @@
 
 ## Accessibility
 
-| Attribute | Applied To | Outcome |
+| Attribute | Applies to | Outcome |
 | -- | -- | -- |
-| `role` or `aria` | `pf-c-popover` |  accessibility notes. |
+| `role="dialog"` | `.pf-c-popover` | Identifies the element that serves as the popover container. **Required**|
+| `aria-labeledby="[id value of .pf-c-popover__header-title]"` | `.pf-c-popover` | Gives the popover an accessible name by referring to the element that provides the dialog title. **Required when .pf-c-popover__header-title is present** |
+| `aria-label="[title of popover]"` | `.pf-c-popover` | Gives the popover an accessible name. **Required when .pf-c-popover__header-title is _not_ present** |
+| `aria-describedby="[id value of applicable content]"` | `.pf-c-popover` | Gives the popover an accessible description by referring to the popover content that describes the primary message or purpose of the dialog. Not used if there is no static text that describes the popover. |
+| `aria-popover="true"` | `.pf-c-popover` | Tells assistive technologies that the windows underneath the current popover are not available for interaction. **Required**|
+| `aria-label="Close Dialog"` | `.pf-c-popover__close .pf-c-button` | Provides an accessible name for the close button as it uses an icon instead of text. **Required**|
 
 
 ## Usage
 
 | Class | Applied To | Outcome |
 | -- | -- | -- |
-| `.class-name-here` | `<tags-here>` |  Outcome and remarks. |
-| Example: `.btn` | `<button>` |  Initiates a button. Always use it with a modifier class. |
+| `.pf-c-popover` | `<div>` |  Creates a popover. |
+| `.pf-c-popover__arrow` | `.pf-c-popover` |  Creates an arrow pointing towards the element the popover describes. ** Required.** |
+| `.pf-c-popover__content` | `.pf-c-popover` |  Creates the body of the popover. ** Required.** |
+| `.pf-c-popover__close` | `.pf-c-popover__content` |  Contains and positions the close icon in the top-right corner of the popover. ** Required.** |
+| `.pf-c-popover__header` | `.pf-c-popover__content` |  The header text area of the popover. |
+| `.pf-c-popover__title` | `.pf-c-popover__header` |  The actual popover title. |
+| `.pf-c-popover__body` | `.pf-c-popover__content` |  The popover's body text. ** Required.** |
+| `.pf-m-left` | `.pf-c-popover` | Puts arrow to the left of the popover's content. |
+| `.pf-m-right` | `.pf-c-popover` | Puts arrow to the right of the popover's content. |
+| `.pf-m-top` | `.pf-c-popover` | Puts arrow on top of the popover's content. |
+| `.pf-m-bottom` | `.pf-c-popover` | Puts arrow on the bottom of the popover's content. |
```

Comment:
These last four rows are accurate. The class `.pf-m-top` is applied to the element with class `.pf-c-popover`, but I think all the other rows (except for the first) are inaccurate. For example, `.pf-c-popover__arrow` is applied to a `<div>` that's a child of `.pf-c-popover`, but it's not applied to `.pf-c-popover` directly.

Reply:
I opened https://github.com/patternfly/patternfly-next/issues/527 to track these changes separately from this PR.